x-pack/plugins/security_solution/server/lists_integration/endpoint/validators/blocklist_validator.ts

```diff
@@ -228,27 +236,27 @@ export class BlocklistValidator extends BaseValidator {
   }
 
   async validatePreDeleteItem(): Promise<void> {
-    await this.validateCanManageEndpointArtifacts();
+    await this.validateHasWritePrivilege();
   }
 
   async validatePreGetOneItem(): Promise<void> {
-    await this.validateCanManageEndpointArtifacts();
+    await this.validateHasReadPrivilege();
   }
 
   async validatePreMultiListFind(): Promise<void> {
-    await this.validateCanManageEndpointArtifacts();
+    await this.validateHasReadPrivilege();
   }
 
   async validatePreExport(): Promise<void> {
-    await this.validateCanManageEndpointArtifacts();
+    await this.validateHasWritePrivilege();
```

Comment:
should this be checking for `read` since its not doing any updates?

Reply:
It was a request from @caitlinbetz. We did the same for the other artifacts.

Reply:
oh. Ok. Its a bit strange - to be able to get/find, but I can't export.

Reply:
Sorry - I think I was confused previously. That makes sense to me. So those with read or all can export? I am good with that, apologies for the confusion!